libs/application/templates/children-residence-change/src/types/index.ts

```diff
@@ -36,9 +36,11 @@ export interface UserInfo {
 export interface ExternalData {
   nationalRegistry: {
     data: NationalRegistry
+    date: string
   }
   userProfile: {
     data: UserInfo
+    date: string
```

Comment:
How does the response date look like? In what format is the date?

Reply:
It is a string and that is the reason I need to use the `parseISO` when formatting the date..